index.html

```diff
@@ -207,64 +208,91 @@ <h2>Git Workflow</h2>
+            <section>
+                <h2>Links for learning more</h2>
+                <p>
+                    <ul>
+                        <li>Documentation for reference - <a href="http://www.git-scm.com/">http://www.git-scm.com/</a> </li>
+                        <li>A simple fast-paced <a href="http://rogerdudler.github.io/git-guide/">guide to git </a>for reference</li>
+                        <li>On fixing problems with commits - <a href="http://sethrobertson.github.io/GitFixUm/fixup.html">highly recommended</a></li>
+                        <li><a href="https://github.com/Roshanjossey/first-contributions">Git tutorials in other languages</a></li>
+                        <li><a href="https://www.git-tower.com/learn/cheat-sheets/git">Git cheat sheet</a></li>
+                        <li>Using <a href="https://www.atlassian.com/git/tutorials/using-branches">git branch</a></li>
+                        <li>Learning <a href="https://blog.gitprime.com/git-rebase-an-illustrated-guide/">Git Rebase</a></li>
+                        <li>Diving deeper into the working of git - <a href="http://www-cs-students.stanford.edu/~blynn/gitmagic/index.html">GitMagic</a></li>
+                        <li>A video to understand how Github works by <a href="http://tiny.cc/gotvideo">Github</a></li>
+                        <li><a href="http://google.com">For all other problems, our common friend.</a></li>
+                    </ul>
                 </p>
             </section>
             <section data-background="http://i.giphy.com/90F8aUepslB84.gif">
-                <h2>That moment when you know the power of Git</h2>
+                <h2>That moment when you realise the power of Git</h2>
+            </section>
+            <section>
+                <h2>Contributors</h2> If you like what you saw, then do thank the contributors :-
+                <p>
+                    <ul style="font-size:75%;">
+                        <li>Sanyam Khurana - Goes on Twitter by <a href="http://www.twitter.com/ErSanyamKhurana">@ErSanyamKhurana</a> | <a href="http://thegeekyway.com/">The Geeky Way</a></li>
+                        <li>Vipul Gupta - available all over the web by <a href="https://twitter.com/vipulgupta2048">@vipulgupta2048</a> | <a href="https://mixstersite.wordpress.com"> Mixster</a></li>
+                        <li>Sandeep Choudhary - available on Github by <a href="https://github.com/Skchoudhary">@Skchoudhary</a></li>
+                        <li>Vishak Sharma - goes on github by <a href="https://github.com/Vishal0203">@Vishal0203</a></li>
+                    </ul>
+                </p>
+                <p> .. or if you didn't like it, then use your newfound knowledge of git to open an issue and contribute here on <a href="https://github.com/CuriousLearner/git-intro/">GitHub</a>.</p>
             </section>
             <section style="text-align: left;">
                 <h1>THE END</h1>
                 <p>
-                    Questions?
```

Comment:
Undo this. 

May be fix email address to be sanyam@sanyamkhurana.com

Reply:
Done, email ID changed.